install/GNUmakefile

```diff
@@ -0,0 +1,212 @@
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2021 Intel Corporation
+
+# The default target of this installation Makefile copies firmware and
+# topology files produced by ./scripts/xtensa-build-all.sh into a tree
+# structure (staging area) identical to /lib/firmware/intel/{sof,sof-tplg}/
+#
+# After inspecting the staging area, you can run "[sudo] make rsync" to
+# rsync it into either a local or remote /lib/firmware/
+# directory, or to a sof-bin clone or to any other destination of your
+# choice configured in the config.mk file.
+
+
+.DEFAULT_GOAL := stage
+.PHONY: clean stage rsync
+.PHONY: signed unsigned ldicts topologies
+.PHONY: compare signed_dummies
+
+# Local configuration
+-include config.mk
+
+
+       ###    platforms   ###
```

Comment:
Btw, I didn't immediately realize these were subheadings. I do think having some visual cues about structure is very welcome.

Reply:
@kv2019i suggested something like `# -------------------` offline, will do.